pilot/pkg/autoregistration/controller.go

```diff
@@ -520,6 +525,43 @@ func sanitizeIP(s string) string {
 	return strings.ReplaceAll(s, ":", "-")
 }
 
+// workloadentryinfo just represents the name/namespace of a workloadentry, the minimum information required
+// for a delete
+type workloadentryinfo struct {
```

Comment:
nit: just use types.NamespacedName which is common. And you cn get it from `curr` with `config.NamespacedName(curr)`